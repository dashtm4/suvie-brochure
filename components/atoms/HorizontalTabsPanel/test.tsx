import { shallow } from 'enzyme'
import toJSON from 'enzyme-to-json'

import { HorizontalTabsPanel } from '.'

describe('<HorizontalTabsPanel />', () => {
  const children = 'Test children'

  it('should renders correct <HorizontalTabsPanel />', () => {
    const wrapper = shallow(
      <HorizontalTabsPanel>{children}</HorizontalTabsPanel>
    )

    expect(wrapper.contains(children)).toBeTruthy()

    expect(toJSON(wrapper)).toMatchSnapshot()
  })
})
