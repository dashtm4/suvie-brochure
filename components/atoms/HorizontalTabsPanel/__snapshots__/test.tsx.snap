// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HorizontalTabsPanel /> should renders correct <HorizontalTabsPanel /> 1`] = `
<TabPanel
  className="react-tabs__tab-panel"
  forceRender={false}
  selectedClassName="react-tabs__tab-panel--selected"
>
  Test children
</TabPanel>
`;
